.circleci/config.pkl

```diff
@@ -157,6 +157,20 @@ jobs {
       }
     }
   }
+  
+  ["test-format"] {
+    docker {
+      new {
+        image = "swift:5.9-rhel-ubi9"
+      }
+    }
+    steps {
+      "checkout"
+      new RunStep {
+        command = "swift package plugin --allow-writing-to-package-directory swiftformat --lint ."
```

Comment:
I think it would be helpful to have this as a target in the `Makefile`

Reply:
Added a target to the makefile!